mopidy/config/types.py

```diff
@@ -258,6 +258,8 @@ def deserialize(self, value, display=False):
         validators.validate_required(value, self._required)
         if not value.strip():
             return None
+        if re.search('/', value):
+            return path.expand_path(value)
```

Comment:
Also, what do you think about requiring Unix domain sockets to start with 'unix:' like apache and nginx configs? This would remove the hard requirement for a slash when you are specifying foo.sock in the current working directory whilst still being unambiguous.